sdk/README.md

```diff
@@ -0,0 +1,6 @@
+# Contrast SDK
+
+**Caution:** This SDK is still under active development and not fit for external use yet.
+Please expect breaking changes with new minor versions.
```

Comment:
One idea I found while searching for ways to highlight instability of APIs: we could have an expressive build tag on the sdk, like `// +build contrast-unstable-api`, to prevent accidental usage.

Reply:
Asked for another review just to be sure, as the tag has to be added to quite a few places.

Reply:
I missed adding the `tags` flag to the `go test` command of the checkPhase in the nix build. Now the tag is only in the sdk's go files. Thanks for pointing it out.